app/controllers/users_controller.rb

```diff
@@ -14,8 +14,29 @@ def create
     end
   end
 
+  def edit
+    @user = User.find(params[:id])
```

Comment:
Use `find_by` and also this line of code is repetitive so let's move this a before action filter